packages/deno/src/integrations/breadcrumbs.ts

```diff
@@ -0,0 +1,171 @@
+import { addBreadcrumb, defineIntegration, getClient } from '@sentry/core';
+import type { Client, Event as SentryEvent, HandlerDataConsole, HandlerDataFetch, IntegrationFn } from '@sentry/types';
+import type { FetchBreadcrumbData, FetchBreadcrumbHint } from '@sentry/types/build/types/breadcrumb';
+import {
+  addConsoleInstrumentationHandler,
+  addFetchInstrumentationHandler,
+  getEventDescription,
+  safeJoin,
+  severityLevelFromString,
+} from '@sentry/utils';
+
+interface BreadcrumbsOptions {
+  console: boolean;
+  fetch: boolean;
+  sentry: boolean;
+}
+
+const INTEGRATION_NAME = 'Breadcrumbs';
+
+/**
+ * This breadcrumbsIntegration is almost the same as the one from @sentry/browser.
+ * The Deno-version does not support browser-specific APIs like dom, xhr and history.
+ */
+const _breadcrumbsIntegration = ((options: Partial<BreadcrumbsOptions> = {}) => {
```

Comment:
l: Let's move the docstring from here to `export const breadcrumbsIntegration` so it shows up in user's IDEs.